src/scripts/main.js

```diff
@@ -1,3 +1,48 @@
 'use strict';
 
-// write code here
+const tbody = document.body.querySelector('tbody');
+const tbodyChildrens = [...document.querySelector('tbody').children];
+const navListElements = [...document.body.querySelector('tr').children];
+
+for (const element of navListElements) {
+  element.addEventListener('click', (action) => {
+    const getTextContent = (item) => {
+      return item.textContent;
+    };
+
+    if (action.target === navListElements[0]) {
```

Comment:
It is better to use [switch...case](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch) if you know all the possible variants of action.taget.
It will make your code cleaner